### ISSUE TITLE
meteor issue

### ISSUE THREAD

**Issue by 520jefferson** [Maluuba/nlg-eval#120]
The meteor don't have the chinese word data, and the meteor mainpage don't have too. 
whether it's not proper to evaluate chinese nlg task using meteor ？

**Comment by kracwarlock** [comment]
Yes, Chinese doesn't seem to be supported by METEOR: https://www.cs.cmu.edu/~alavie/METEOR/README.html. Some of the other metrics would work but we don't provide tokenizers for Chinese. If you can handle that you should be able to make the other metrics work.
